### PR TITLE
_post: RIP pullmoll

### PR DESCRIPTION
**Pull request by Gottox**
I'm deeply shocked and sorry to share this message.

Please, if anyone finds the better words than I did, feel free to share you're changes. He deserved courtesy.

### CODE REVIEW

File: _posts/2021-03-27-RIP-pullmoll.markdown
Comment (ericonr) [inline _posts/2021-03-27-RIP-pullmoll.markdown:11]:
```suggestion
Today we were informed that our long time contributor and maintainer Jürgen Buchmüller (pullmoll) passed away.
```

**Comment by Gottox** [comment]
Collaborative editor for the news entry: https://demo.hedgedoc.org/UIHyW8N_Q82pI-Ziv2d6Kw#

ToDo:

* [ ] mention his financial involvement
* [ ] meintion a few of his milestones for the project.

File: _posts/2021-03-27-RIP-pullmoll.markdown
Comment (ericonr) [inline _posts/2021-03-27-RIP-pullmoll.markdown:14]:
```suggestion
**Thank you, pullmoll.**
```

**Comment by Gottox** [comment]
@void-linux/pkg-committers please review this. I plan to publish the news item tomorrow.

File: _posts/2021-03-27-RIP-pullmoll.markdown
Comment (leahneukirchen) [inline _posts/2021-03-27-RIP-pullmoll.markdown:15]:
```
financially, too.
```

File: _posts/2021-03-27-RIP-pullmoll.markdown
Comment (leahneukirchen) [inline _posts/2021-03-27-RIP-pullmoll.markdown:17]: `invested`

File: _posts/2021-03-27-RIP-pullmoll.markdown
Comment (leahneukirchen) [inline _posts/2021-03-27-RIP-pullmoll.markdown:18]: `s/cared about/took care of/`

**Review by leahneukirchen — CHANGES_REQUESTED** [review]

File: _posts/2021-03-27-RIP-pullmoll.markdown
Comment (ericonr) [inline _posts/2021-03-27-RIP-pullmoll.markdown:22]: people instead of person, suggested by ifreund

File: _posts/2021-03-27-RIP-pullmoll.markdown
Comment (ericonr) [inline _posts/2021-03-27-RIP-pullmoll.markdown:4]: What does this do?

Also, remember to squash the commits together before merging.

**Review by the-maldridge — APPROVED** [review]
I would make the change of person/people noted below, as it flows smoother and is more typical in this style of writing.  Other than that, I have no feedback.

**Review by ahesford — APPROVED** [review]
Could consider a title change from "RIP pullmoll" to "In memoriam pullmoll", but that's an arbitrary style change.

File: _posts/2021-03-27-RIP-pullmoll.markdown
Comment (sgn) [inline _posts/2021-03-27-RIP-pullmoll.markdown:15]: `s/Void /Void Linux/`, too.

File: _posts/2021-03-27-RIP-pullmoll.markdown
Comment (ericonr) [inline _posts/2021-03-27-RIP-pullmoll.markdown:19]:
```suggestion
He invested lots of his time in keeping GCC up to date and well tested, supplied tons of
patches to port packages to new compilers and platforms and also took care of
updating many packages, including several non-trivial ones, like OpenJDK and LibreOffice.
```

File: _posts/2021-03-27-RIP-pullmoll.markdown
Comment (ericonr) [inline _posts/2021-03-27-RIP-pullmoll.markdown:24]: I'd like to leave something to friends and family. Maybe it fits here?

> Our thoughts and condolences go out to Jürgen's family, friends, and loved ones.

**Review by ericonr — COMMENTED** [review]
Mostly a nit, but (at least for me) keeping the profile image colored makes it a bit more optimistic (I'm not sure it's the right word), so if you agree, I think we could use the original pfp instead...

I'm not sure how to express it, sorry.

File: _posts/2021-03-27-RIP-pullmoll.markdown
Comment (Gottox) [reply _posts/2021-03-27-RIP-pullmoll.markdown:19]: This is much better, thank you.

File: _posts/2021-03-27-RIP-pullmoll.markdown
Comment (Gottox) [reply _posts/2021-03-27-RIP-pullmoll.markdown:22]: applied

File: _posts/2021-03-27-RIP-pullmoll.markdown
Comment (Gottox) [reply _posts/2021-03-27-RIP-pullmoll.markdown:24]: applied

File: _posts/2021-03-27-RIP-pullmoll.markdown
Comment (Gottox) [reply _posts/2021-03-27-RIP-pullmoll.markdown:15]: applied

**Comment by Anachron** [comment]
Very sad to read this. 

Should be update [acknowledgments](https://github.com/void-linux/void-linux.github.io/blob/fa91f23474e8a7fb72540b9d4aa74ee7dfb5a491/acknowledgments/index.md#L33) as well?

**Comment by Gottox** [comment]
Yes, we actually should.

**Pull request merged**
